src/eth1spec/spec.py

```diff
@@ -240,6 +230,23 @@ def verify_transaction(tx: Transaction) -> bool:
     verified : `bool`
         True if the transaction can be executed, or False otherwise.
     """
+    return intrinsic_cost(tx) <= tx.gas
+
+
+def intrinsic_cost(tx: Transaction) -> Uint:
+    """
+    Verifies a transaction.
```

Comment:
Need to update the docs for this function.

Reply:
oof nice catch